src/webgpu/api/validation/encoding/queries/resolveQuerySet.spec.ts

```diff
@@ -10,14 +10,47 @@ export const g = makeTestGroup(ValidationTest);
 
 export const kQueryCount = 2;
 
+g.test('resolveQuerySet,invalid_queryset_and_destination_buffer')
+  .desc(
+    `
+Tests that resolve query set with invalid object.
+- invalid GPUQuerySet that failed during creation.
+- invalid destination buffer that failed during creation.
+  `
+  )
+  .subcases(
+    () =>
+      [
+        { querySetState: 'invalid', destinationState: 'valid' },
+        { querySetState: 'valid', destinationState: 'invalid' },
```

Comment:
Please also test with both being valid, as a control case, to make sure the test code is valid aside from the querySetState/destinationState.

Reply:
The test with both being valid should be same with the control case in `resolveQuerySet,first_query_and_query_count`. Do we need to add control cases for each `cases` or `subcases`, even they have been tested?

Reply:
We should, yes. The point of the control case is not to the test the browser, but to test the test - that is, make sure there aren't any other mistakes in the test function (and especially that no mistakes get introduced in the future).